closed/custom/modules/java.base/Copy.gmk

```diff
@@ -99,7 +99,7 @@ $(call openj9_copy_shlibs, \
 	omrsig \
 	)
 
-ifeq (zos,$(OPENJDK_TARGET_OS))
+ifeq ($(call isTargetOs, zos), true)
 
```

Comment:
While you're here, please remove blank line 103 (for consistency - notice there's no blank line after line 33).